docs/custom-images-contract.rst

```diff
@@ -24,15 +24,33 @@ Contract statements
 
     docker run myscrapyimage start-crawl
 
-2. Docker image should be able to return a spiders list via ``list-spiders`` command without arguments.
-   ``list-spiders`` should be :ref:`executable and located on the search path <scripts-example>`.
+2. Docker image should be able to return its metadata via ``shub-image-info`` command without arguments.
+   ``shub-image-info`` should be :ref:`executable and located on the search path <scripts-example>`.
+   For now only ``project_type`` and ``spiders`` fields are supported.
 
 .. code-block:: bash
 
-   docker run myscrapyimage list-spiders
+   docker run myscrapyimage shub-image-info
+
+    {
+        "project_type": "casperjs",
```

Comment:
It should be clear which types we're going to support with the protocol. 
I know it's `scrapy` and `casperjs` for sure, any others to include to the doc?